internal/cmd/users.go

```diff
@@ -280,19 +280,22 @@ func updateUser(cli *CLI, name string) error {
 			return err
 		}
 
-	PROMPT:
+		for {
+			req.Password, err = promptSetPassword(cli, req.OldPassword)
+			if err != nil {
+				return err
+			}
 
-		req.Password, err = promptSetPassword(cli, req.OldPassword)
-		if err != nil {
-			return err
-		}
+			logging.Debugf("call server: update user %s", req.ID)
```

Comment:
```suggestion
```
not related to this PR, but might be worth cleaning this up while in the neighborhood